docs/changelog.md

```diff
@@ -10,6 +10,18 @@ We use _breaking :warning:_ to mark changes that are not backward compatible \(r
 
 ## Unreleased
```

Comment:
Don't think we need this title anymore, it looks kind of strange when looking at the docs.